src/libmongoc/tests/test-mongoc-client-session.c

```diff
@@ -3013,13 +3036,14 @@ test_session_install (TestSuite *suite)
                                        test_sessions_spec_cb,
                                        test_framework_skip_if_no_sessions);
 
-   TestSuite_AddFull (suite,
-                      "/Session/dirty",
-                      test_session_dirty,
-                      NULL /* dtor */,
-                      NULL /* ctx */,
-                      test_framework_skip_if_no_sessions,
-                      test_framework_skip_if_no_failpoint,
-                      /* Tests with retryable writes, requires non-standalone. */
-                      test_framework_skip_if_single);
+   TestSuite_AddFull (
+      suite,
+      "/Session/dirty",
+      test_session_dirty,
+      NULL /* dtor */,
+      NULL /* ctx */,
+      test_framework_skip_if_no_sessions,
+      test_framework_skip_if_no_failpoint,
+      /* Tests with retryable writes, requires non-standalone. */
+      test_framework_skip_if_single);
```

Comment:
As a note, we do not (yet) have automated lint tasks. So sometimes clang-format of a file will inadvertently change code you did not modify. This looks like the right output for clang-format 4.0.1.